src/v/utils/uuid.cc

```diff
@@ -0,0 +1,7 @@
+#include "utils/uuid.h"
```

Comment:
Copyright banner?

(for protecting our red hot `operator<<` IP...

Reply:
thanks. missed that